heron/spi/src/java/com/twitter/heron/spi/utils/PackingTestUtils.java

```diff
@@ -72,15 +76,28 @@ public static PackingPlan testPackingPlan(String topologyName, IPacking packing)
     return testContainerPlan(containerId, 0, 1);
   }
 
+  @SuppressWarnings("unchecked")
   public static PackingPlan.ContainerPlan testContainerPlan(int containerId,
                                                             Integer... instanceIndices) {
+    List<Pair<String, Integer>> instanceInfo = new ArrayList<>();
+    for (int instanceIndex : instanceIndices) {
+      String componentName = "componentName-" + instanceIndex;
+      instanceInfo.add(new Pair<>(componentName, instanceIndex));
+    }
+    return testContainerPlan(containerId, instanceInfo.toArray(
+        (Pair<String, Integer>[]) Array.newInstance(Pair.class, instanceInfo.size())));
+  }
+
+  @SafeVarargs
+  public static PackingPlan.ContainerPlan testContainerPlan(int containerId,
+                                                            Pair<String, Integer>...
```

Comment:
Last nit, is that you typically don't want to split the type from the variable on two separate lines.